integration-tests/step-functions/aws/nino_issue_credential/nino_issue_credential.test.ts

```diff
@@ -452,6 +452,7 @@ describe("Nino Check Hmrc Issue Credential", () => {
     clientSessionId: "252561a2-c6ef-47e7-87ab-93891a2a6a41",
     persistentSessionId: "156714ef-f9df-48c2-ada8-540e7bce44f7",
     evidenceRequest,
+    txn: "2bba5d6e-dc56-4291-b1ed-4f54d263ea63"
```

Comment:
There's some `txn: expect.any(String),` in this file do you think we could change them to expect a proper value now?